07:17:46 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:17:46 [Info] Gateway: Connecting
07:17:46 [Info] Gateway: Connected
07:17:47 [Info] Gateway: Ready
07:18:19 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:18:19 [Info] Gateway: Connecting
07:18:20 [Info] Gateway: Connected
07:18:21 [Info] Gateway: Ready
07:18:23 [Msg] Successful command 'help' executed by user: 'ryxx'.
07:19:06 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:19:06 [Info] Gateway: Connecting
07:19:07 [Info] Gateway: Connected
07:19:07 [Info] Gateway: Ready
07:23:12 [Warning] Gateway: System.Exception: WebSocket connection was closed
 ---> Discord.Net.WebSocketClosedException: The server sent close 4003: "Not authenticated."
   at Discord.Net.WebSockets.DefaultWebSocketClient.RunAsync(CancellationToken cancelToken)
   --- End of inner exception stack trace ---
   at Discord.ConnectionManager.<>c__DisplayClass29_0.<<StartAsync>b__0>d.MoveNext()
07:23:12 [Info] Gateway: Disconnecting
07:23:12 [Info] Gateway: Disconnected
07:23:13 [Info] Gateway: Connecting
07:23:13 [Warning] Gateway: Failed to resume previous session
07:23:13 [Info] Gateway: Connected
07:23:14 [Info] Gateway: Ready
07:23:50 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:23:50 [Info] Gateway: Connecting
07:24:34 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:24:34 [Info] Gateway: Connecting
07:24:35 [Info] Gateway: Connected
07:24:36 [Info] Gateway: Ready
07:27:02 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:27:02 [Info] Gateway: Connecting
07:27:02 [Verbose] Rest: GET gateway: 275 ms
07:27:03 [Info] Gateway: Connected
07:27:03 [Verbose] Gateway: Connected to Jesses Scams
07:27:03 [Verbose] Gateway: Connected to Slick Rentals
07:27:03 [Info] Gateway: Ready
07:27:08 [Verbose] Rest: POST channels/831283546999095316/messages: 241.89 ms
07:29:46 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:29:46 [Info] Gateway: Connecting
07:29:46 [Verbose] Rest: GET gateway: 233.54 ms
07:29:47 [Info] Gateway: Connected
07:29:47 [Verbose] Gateway: Connected to Jesses Scams
07:29:47 [Verbose] Gateway: Connected to Slick Rentals
07:29:47 [Info] Gateway: Ready
07:29:55 [Verbose] Rest: POST channels/745666110803935353/messages: 256.05 ms
07:33:03 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:33:03 [Info] Gateway: Connecting
07:33:03 [Verbose] Rest: GET gateway: 235.81 ms
07:33:04 [Info] Gateway: Connected
07:33:04 [Verbose] Gateway: Connected to Jesses Scams
07:33:04 [Verbose] Gateway: Connected to Slick Rentals
07:33:04 [Info] Gateway: Ready
07:33:11 [Verbose] Rest: POST channels/745666110803935353/messages: 226.56 ms
07:33:11 [Verbose] Rest: POST channels/745666110803935353/messages: 178.51 ms
07:33:11 [Msg] Successful command 'update' executed by user: 'ryxx'.
07:33:11 [Verbose] Command: Executed "update" for ryxx#0001 in Slick Rentals/bot-commands
07:34:25 [Info] Discord: Discord.Net v3.0.0-dev-20210529.2 (API v9)
07:34:25 [Info] Gateway: Connecting
07:34:25 [Verbose] Rest: GET gateway: 208.7 ms
07:34:26 [Info] Gateway: Connected
07:34:26 [Verbose] Gateway: Connected to Slick Rentals
07:34:26 [Verbose] Gateway: Connected to Jesses Scams
07:34:26 [Info] Gateway: Ready
07:34:34 [Verbose] Rest: POST channels/745666110803935353/messages: 234.81 ms
07:34:34 [Error] Command: Discord.Commands.CommandException: Error occurred executing "update" for ryxx#0001 in Slick Rentals/bot-commands.
 ---> System.InvalidOperationException: RunSynchronously may not be called on a task not bound to a delegate, such as the task returned from an asynchronous method.
   at System.Threading.Tasks.Task.InternalRunSynchronously(TaskScheduler scheduler, Boolean waitForCompletion)
   at SlickRentals_Discord_Bot.Modules.Utility.Update()
   at Discord.Commands.ModuleClassBuilder.<>c__DisplayClass6_0.<<BuildCommand>g__ExecuteCallback|0>d.MoveNext()
--- End of stack trace from previous location ---
   at Discord.Commands.CommandInfo.ExecuteInternalAsync(ICommandContext context, Object[] args, IServiceProvider services)
   --- End of inner exception stack trace ---
07:34:34 [Msg] Invalid command attempted by user:'ryxx' RunSynchronously may not be called on a task not bound to a delegate, such as the task returned from an asynchronous method.
07:34:34 [Verbose] Command: Executed "update" for ryxx#0001 in Slick Rentals/bot-commands
